app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -36,9 +62,20 @@ def destroy
       end
 
       def find_parent_config_template
-        @config_template = ConfigTemplate.authorized(:view_templates).find(params[:config_template_id])
-        not_found unless @config_template
-        @config_template
+        @config_template = ConfigTemplate.authorized(:view_templates).find(params[:config_template_id]) rescue nil
+      end
+
+      def requested_resource
```

Comment:
It is necessary, the subtle difference is that we'd like to return one of config_template OR nested_obj as the resource for the template combination (tests fail w/o assigning this resource).